packages/block-editor/src/components/block-navigation/dropdown.js

```diff
@@ -32,6 +32,19 @@ function BlockNavigationDropdownToggle( { isEnabled, onToggle, isOpen } ) {
 			isDisabled: ! isEnabled,
 		}
 	);
+
+	const contentWindow = window.frames[ 'editor-content' ];
+
+	useShortcut(
+		'core/edit-post/toggle-block-navigation',
+		useCallback( onToggle, [ onToggle ] ),
+		{
+			bindGlobal: true,
+			isDisabled: ! isEnabled || ! contentWindow,
+			altWindow: contentWindow,
```

Comment:
Shouldn't shortcuts work across windows?

Reply:
Yes, I've fixed them in different ways so far, at then end I'll unify it and make it work across windows. First priority is making all the tests pass and just making it work. 😄